provider/go.go

```diff
@@ -108,13 +108,24 @@ func (p *Go) BoolGetter() func() (bool, error) {
 }
 
 func (p *Go) paramAndEval(param string, val any) error {
+	if str, ok := val.(string); ok {
+		val = "\"" + str + "\""
+	}
+
 	_, err := p.vm.Eval(fmt.Sprintf("%s := %v;", param, val))
+	if err == nil {
```

Comment:
Warum brauchts das? Schien mir in go egtl. plausibel. Gibts ggf. ein "assign" Kommando in Yaegi?

Reply:
Ich habe nichts gefunden, um variable direkt zu setzen, deshalb der Spass mit dem zusammenbauen der Variablendeklaration.

Das setzen von "param" und "val" ist eigentlich nicht nötig, aber es ist im Javascript Plugin so drin, deshalb habe ich das hier nachgezogen, so dass die Implementierungen möglichst gleich sind.

Reply:
Hab das erstmal aus dem PR rausgenommen